auth-web/src/assets/scss/base.scss

```diff
@@ -131,6 +131,10 @@ legend {
   }
 }
 
+.v-btn:hover {
+  opacity: .8;
```

Comment:
I'm a bit concerned about this style globally, as it will affect all buttons in this UI, and isn't consistent with Filings UI or Create UI.
Is there a special case you're targeting, and could that be targeted more precisely?

Reply:
Yea, i was on the fence with this. According to the invision style guides. All the buttons in our future state will be .8 op on hover. Government standard apparently. Though, i certainly dont doing this individually, would you prefer that?

Reply:
My first concern is that this base style will affect all buttons, which we may want to do. My second concern is, if this is a standard, then we should be doing it for the other UIs also (but not willy-nilly -- will need regression testing).
I'm unsure of the best approach here... maybe to keep risk low, just apply this style where it's needed? That may result in a bit of tech debt but oh well, MVP is close!

Reply:
Totally agree :) Updated in next commit!